auth/kerberos/src/krb.cpp

```diff
@@ -798,51 +798,37 @@ std::list<std::string> renew_kerberos_tickets_domainless(std::string krb_files_d
         {
             std::pair<int, std::string> gmsa_ticket_result;
             std::string krb_cc_name = krb_ticket->krb_file_path;
-            // check if the ticket is ready for renewal
-            if ( is_ticket_ready_for_renewal( krb_cc_name ) )
+            int num_retries = 1;
+            for ( int i = 0; i <= num_retries; i++ )
             {
-                int num_retries = 1;
-                for ( int i = 0; i <= num_retries; i++ )
+                gmsa_ticket_result = get_gmsa_krb_ticket( krb_ticket->domain_name,
+                                                              krb_ticket->service_account_name,
+                                                              krb_cc_name, cf_logger );
+                if ( gmsa_ticket_result.first != 0 )
                 {
-                    gmsa_ticket_result = get_gmsa_krb_ticket(
-                        krb_ticket->domain_name, krb_ticket->service_account_name,
-                        krb_cc_name, cf_logger );
-                    if ( gmsa_ticket_result.first != 0 )
+                    cf_logger.logger( LOG_ERR, "ERROR: Cannot get gMSA krb ticket" );
```

Comment:
Add retry number in log like "[%d] : ERROR/Warning: Cannot get .." and also add log when ticket creation succeeds also on failure after retries